src/components/FeatureViews/Courses/CourseManagementContainer.js

```diff
@@ -216,18 +205,24 @@ const ClassListItem = ({
                             }}
                         />
                     </Grid>
+                    <Grid item xs={10} sm={9} md={10}>
+                        <Typography
+                            variant="h3"
+                            align="left"
+                            style={{ marginLeft: '.85em' }}
+                        >
+                            {title}
+                        </Typography>
+                    </Grid>
+
                     <Grid
                         item
                         xs={3}
                         sm={4}
                         md={2}
                         className={classes.displayCardMargins}
                     >
-                        <Typography
-                            variant="body1"
-                            align="left"
-                            style={{ marginLeft: '1em' }}
-                        >
+                        <Typography variant="body1" align="left">
                             <span
                                 className={classes.highlightName}
                             >{`${concatFullName}`}</span>
```

Comment:
you didn't write this but since you need to clean out the console.log - can you just take `concatFullName` out? so it's just `<span ...> {concatFullName} </span>`